examples/private-public-ingress/README.md

```diff
@@ -0,0 +1,40 @@
+# Amazon EKS Private and Public Ingress example
+
+This example demonstrates how to provision an Amazon EKS cluster with two nginx ingress controllers; one to expose applications publicaly and the other to expose applications internally. It also assigns security groups to the Network Load Balancers used to expose the internal and external ingress controllers.
+
+This solutions:
+* Deploys Amazon EKS, with 1 Managed Node Group using the Bottlerocket Amazon EKS Optimized AMI spread accross 3 availability zones.
+* Installs the AWS Load Balancer controller for creating Network Load Balancers and Application Load Balancers. This is the recommended approach instead of the built-in AWS cloud provider load balancer controller.
```

Comment:
```suggestion
* Installs the AWS Load Balancer controller for creating Network Load Balancers and Application Load Balancers. This is the recommended approach instead of the in-tree AWS cloud provider load balancer controller.
```